msteams-platform/sbs-teams-connectors.yml

```diff
@@ -66,21 +66,21 @@ items:
         1. In the left panel, select **Show All**.
         1. Select **Teams**.
  
-           :::image type="content" source="./assets/images/meeting-token-generator/meeting-token-generator-admincenter.png" alt-text="Admin center menu":::
+           :::image type="content" source="./assets/images/meeting-token-generator/meeting-token-generator-admincenter.png" alt-text="Screenshot shows the Admin center menu.":::
 
     1.  In the left panel, select **Teams apps**.
        
     1.  Select **Setup policies**.
  
-           :::image type="content" source="~/assets/images/meeting-token-generator/meeting-token-generator-setuppolicies.png" alt-text="Setup policies":::
+           :::image type="content" source="~/assets/images/meeting-token-generator/meeting-token-generator-setuppolicies.png" alt-text="Screenshot shows the setup policies.":::
    
     1.  Select **Global**.
 
-          ![Global](~/assets/images/sbs-connector/Global.png)
-
+          :::image type="content" source="~/assets/images/sbs-connector/Global.png" alt-text="Screenshot shows the steps to select Global.":::
+          
     1.   Toggle **Upload custom apps** to **On** position.
 
-          :::image type="content" source="~/assets/images/meeting-token-generator/meeting-token-generator-toggleOn.png" alt-text="Toggle On":::
+          :::image type="content" source="~/assets/images/meeting-token-generator/meeting-token-generator-toggleOn.png" alt-text="Screenshot shows the Toggle On position.":::
```

Comment:
Re-write this to mention what's the setting and not the on/off switch of the setting.